app/uk/gov/hmrc/plasticpackagingtaxregistrationfrontend/config/ErrorHandler.scala

```diff
@@ -9,9 +25,13 @@ import uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler
 import uk.gov.hmrc.plasticpackagingtaxregistrationfrontend.views.html.error_template
 
 @Singleton
-class ErrorHandler @Inject()(error_template: error_template, val messagesApi: MessagesApi)(implicit appConfig: AppConfig)
-    extends FrontendErrorHandler {
+class ErrorHandler @Inject() (error_template: error_template, val messagesApi: MessagesApi)(implicit
+  appConfig: AppConfig
```

Comment:
AppConfig shouldn't be as implicit injected into classes/controllers.

AppConfig should be in view constructor and only in govuk_wrapper, so you should be able to remove a lot of unnecessary inject.

I think it's worth to mention to the team which prepared this template, because it's redundant.

Reply:
I think it's similar with `Messages` or `MessagesApi`